### PR TITLE
Docs: corrects aliases

### PR DESCRIPTION
**Pull request by chri2547**
This PR corrects a number of aliases in user management docs.

### CODE REVIEW

File: docs/sources/administration/manage-organizations/_index.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ description: 'Describes how to use organizations to isolate dashboard to users a
 weight: 300
 aliases:
   [
-    docs/grafana/latest/manage-users/server-admin/_index.md,
-    docs/grafana/latest/manage-users/server-admin/server-admin-manage-orgs.md,
+    /docs/grafana/latest/manage-users/server-admin/,
```
Comment (achatterjee-grafana) [inline docs/sources/administration/manage-organizations/_index.md:8]:
```suggestion
    "/docs/grafana/latest/manage-users/server-admin/",
```

File: docs/sources/administration/manage-organizations/_index.md [hunk 2]
```diff
@@ -5,8 +5,8 @@ description: 'Describes how to use organizations to isolate dashboard to users a
...
+    /docs/grafana/latest/manage-users/server-admin/server-admin-manage-orgs/,
```
Comment (achatterjee-grafana) [inline docs/sources/administration/manage-organizations/_index.md:9]:
```suggestion
    "/docs/grafana/latest/manage-users/server-admin/server-admin-manage-orgs/",
```

File: docs/sources/enterprise/access-control/rbac-fixed-basic-role-definitions.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: 'RBAC role definitions'
 menuTitle: 'RBAC role definitions'
 description: 'This topic includes a table that lists permission associated with Grafana fixed and basic roles.'
-aliases: [docs/grafana/latest/enterprise/access-control/fine-grained-access-control-references/]
+aliases: [/docs/grafana/latest/enterprise/access-control/fine-grained-access-control-references/]
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/access-control/rbac-fixed-basic-role-definitions.md:5]:
```suggestion
aliases: ["/docs/grafana/latest/enterprise/access-control/fine-grained-access-control-references/"]
```

File: docs/sources/enterprise/access-control/plan-rbac-rollout-strategy.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 title: 'Plan your Grafana RBAC rollout strategy'
 menuTitle: 'Plan your RBAC rollout strategy'
 description: 'Plan your RBAC rollout strategy before you begin assigning roles to users and teams.'
-aliases: [docs/grafana/latest/enterprise/access-control/usage-scenarios/]
+aliases: [/docs/grafana/latest/enterprise/access-control/usage-scenarios/]
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/access-control/plan-rbac-rollout-strategy.md:5]:
```suggestion
aliases: ["/docs/grafana/latest/enterprise/access-control/usage-scenarios/"]
```

File: docs/sources/enterprise/access-control/manage-rbac-roles.md [hunk 5]
```diff
@@ -4,8 +4,8 @@ menuTitle: 'Manage RBAC roles'
 description: 'Learn how to view permissions associated with roles, create custom roles, and update and delete roles in Grafana.'
 aliases:
   [
-    docs/grafana/latest/enterprise/access-control/manage-role-assignments/,
-    docs/grafana/latest/enterprise/access-control/provisioning/,
+    /docs/grafana/latest/enterprise/access-control/manage-role-assignments/,
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/access-control/manage-rbac-roles.md:7]: Add the (") where appropriate following this example:
aliases = ["/docs/grafana/latest/manage-users/", "/docs/grafana/latest/manage-users/server-admin/", "/docs/grafana/latest/manage-users/server-admin/server-admin-manage-users/"]

File: docs/sources/enterprise/access-control/custom-role-actions-scopes.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 title: 'Grafana RBAC permissions, actions, and scopes'
 menuTitle: 'RBAC permissions, actions, and scopes'
 description: 'Learn about Grafana RBAC permissions, actions, and scopes.'
-aliases: [docs/grafana/latest/enterprise/access-control/permissions/]
+aliases: [/docs/grafana/latest/enterprise/access-control/permissions/]
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/access-control/custom-role-actions-scopes.md:5]:
```suggestion
aliases: ["/docs/grafana/latest/enterprise/access-control/permissions/"]
```

File: docs/sources/enterprise/access-control/assign-rbac-roles.md [hunk 7]
```diff
@@ -4,8 +4,8 @@ menuTitle: 'Assign RBAC roles'
 description: 'Learn how to assign RBAC roles to users and teams in Grafana.'
 aliases:
   [
-    docs/grafana/latest/enterprise/access-control/manage-role-assignments/manage-user-role-assignments/,
-    docs/grafana/latest/enterprise/access-control/manage-role-assignments/manage-built-in-role-assignments/,
+    /docs/grafana/latest/enterprise/access-control/manage-role-assignments/manage-user-role-assignments/,
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/access-control/assign-rbac-roles.md:7]: Needs the (")

**Review by achatterjee-grafana — COMMENTED** [review]
Missing " in some examples. 

Need backporting to v8.5.x to have aliases available in latest.

File: docs/sources/enterprise/access-control/assign-rbac-roles.md (hunk 7 above)
Comment (chri2547) [reply docs/sources/enterprise/access-control/assign-rbac-roles.md:7]: got it, thanks.

File: docs/sources/enterprise/access-control/manage-rbac-roles.md (hunk 5 above)
Comment (chri2547) [reply docs/sources/enterprise/access-control/manage-rbac-roles.md:7]: good catch, thanks!

File: docs/sources/enterprise/access-control/plan-rbac-rollout-strategy.md [hunk 8]
```diff
@@ -2,7 +2,7 @@
 title: 'Plan your Grafana RBAC rollout strategy'
 menuTitle: 'Plan your RBAC rollout strategy'
 description: 'Plan your RBAC rollout strategy before you begin assigning roles to users and teams.'
-aliases: [docs/grafana/latest/enterprise/access-control/usage-scenarios/]
+aliases: ['/docs/grafana/latest/enterprise/access-control/usage-scenarios/']
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/access-control/plan-rbac-rollout-strategy.md:5]:
```suggestion
aliases: ["/docs/grafana/latest/enterprise/access-control/usage-scenarios/"]
```

File: docs/sources/enterprise/access-control/rbac-fixed-basic-role-definitions.md [hunk 9]
```diff
@@ -2,7 +2,7 @@
 title: 'RBAC role definitions'
 menuTitle: 'RBAC role definitions'
 description: 'This topic includes a table that lists permission associated with Grafana fixed and basic roles.'
-aliases: [docs/grafana/latest/enterprise/access-control/fine-grained-access-control-references/]
+aliases: ['/docs/grafana/latest/enterprise/access-control/fine-grained-access-control-references/']
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/access-control/rbac-fixed-basic-role-definitions.md:5]:
```suggestion
aliases: ["/docs/grafana/latest/enterprise/access-control/fine-grained-access-control-references/"]
```

File: docs/sources/administration/manage-organizations/_index.md [hunk 10]
```diff
@@ -5,8 +5,8 @@ description: 'Describes how to use organizations to isolate dashboard to users a
 weight: 300
 aliases:
   [
-    docs/grafana/latest/manage-users/server-admin/_index.md,
-    docs/grafana/latest/manage-users/server-admin/server-admin-manage-orgs.md,
+    '/docs/grafana/latest/manage-users/server-admin/',
+    '/docs/grafana/latest/manage-users/server-admin/server-admin-manage-orgs/',
```
Comment (achatterjee-grafana) [inline docs/sources/administration/manage-organizations/_index.md:9]:
```suggestion
    "/docs/grafana/latest/manage-users/server-admin/server-admin-manage-orgs/",
```

**Review by achatterjee-grafana — COMMENTED** [review]
Found a few single quotes, sorry if I had added them. Looks good to me, but double-check one more time if you like, just to make sure. Thanks!

**Review by achatterjee-grafana — APPROVED** [review]

**Pull request merged**